src/StreamJsonRpc/JsonRpcEventSource.cs

```diff
@@ -278,18 +278,33 @@ internal static string GetArgumentsString(object? arguments)
 
             const int maxLength = 128;
 
-            if (arguments is object?[] args)
+            if (arguments is null)
+            {
+                return string.Empty;
+            }
+            else if (arguments is object[] args)
```

Comment:
It looks like you inadvertently dropped the `?` in `object?[]` here.

Reply:
Oops